gef-ui/server/utils/helpers.test.js

```diff
@@ -607,9 +623,16 @@ describe('mapSummaryList()', () => {
         user: MOCK_REQUEST,
       };
 
-      const { text } = mapSummaryList(MOCK_ISSUED_FACILITY, mockedDisplayItems, mapSummaryParams, true)[0].actions.items[0];
+      const actionItems = mapSummaryList(MOCK_ISSUED_FACILITY, mockedDisplayItems, mapSummaryParams, true)[0].actions.items;
       // should be blank so cannot change
```

Comment:
Is this comment no longer relevant? Or if it's still relevant maybe a rewording would be good as I'm not sure I understand it